integration/token/fungible/views/redeem.go

```diff
@@ -21,6 +22,8 @@ import (
 
 // Redeem contains the input information for a redeem
 type Redeem struct {
+	// Auditor is the name of the auditor that must be contacted to approve the operation
```

Comment:
name vs. enrolment ID?

Reply:
not the EID but just the name/label used in the configuration to identify the FSC auditor identity